### PR TITLE
Add dictionary to json project

### PR DESCRIPTION
**Pull request by tanuj208**
Added json.dict, a dictionary to create json inputs and updated build and dockerfile scripts for the same.

### CODE REVIEW

File: projects/json/parse_afl_fuzzer.options [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[libfuzzer]
+max_len = 456
+timeout = 10
```
Comment (oliverchang) [inline projects/json/parse_afl_fuzzer.options:3]: why are these needed? specifying max_len in .options file is also no longer recommended. Similarly, why do we specify timeout?

just name the .dict parse_afl_fuzzer.dict and it'll automatically get used.

Comment (tanuj208) [reply projects/json/parse_afl_fuzzer.options:3]: I added max_len and timeout because they were there in their previous [.options](https://github.com/google/oss-fuzz/blob/master/projects/json/fuzzer-parse.options) file.
I'll remove them
